<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="ImageWidget" name="img_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ImageWidget" name="img_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="cbxImg0">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSuppress">
           <property name="text">
            <string>Suppress</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbxImg1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btnImage0">
          <property name="text">
           <string>Favorite</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="btnPass">
            <property name="text">
             <string>Pass</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDraw">
            <property name="text">
             <string>Draw</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnImage1">
          <property name="text">
           <string>Favorite</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionGenerate"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuUsers">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Users</string>
    </property>
    <widget class="QMenu" name="menuSelect">
     <property name="title">
      <string>Select</string>
     </property>
    </widget>
    <addaction name="actionAdd"/>
    <addaction name="menuSelect"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuUsers"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="iconText">
    <string>Save as...</string>
   </property>
   <property name="toolTip">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="text">
    <string>Generate...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QLabel</extends>
   <header>imagewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnImage1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectImage1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnImage0</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectImage0()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDraw</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>draw()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSuppress</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>suppress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPass</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>passImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectImage1()</slot>
  <slot>selectImage0()</slot>
  <slot>draw()</slot>
  <slot>passImages()</slot>
  <slot>suppress()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
